src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsremandandsentencingapi/jpa/entity/SentenceEntity.kt

```diff
@@ -163,7 +163,19 @@ class SentenceEntity(
     return sentenceEntity
   }
 
-  fun copyFrom(sentence: LegacyCreateSentence, createdBy: String, consecutiveTo: SentenceEntity?, isManyCharges: Boolean): SentenceEntity {
+  fun copyFrom(
+    sentence: LegacyCreateSentence,
+    createdBy: String,
+    consecutiveTo: SentenceEntity?,
+    isManyCharges: Boolean,
+  ): SentenceEntity {
+    val newSentenceServeType =
+      when {
+        consecutiveTo != null -> "CONSECUTIVE"
+        this.sentenceServeType == "FORTHWITH" -> "FORTHWITH"
+        else -> "CONCURRENT"
+      }
```

Comment:
not sure if there's any scenario where the legacy update should update a FORTHWITH to a CONCURRENT, if so the above won't work! and we'll need to rethink